lib/rspec/bash/stubbed_env.rb

```diff
@@ -2,78 +2,124 @@
 require 'English'
```

Comment:
There are a lot of non-SOLID things going on here, especially with the passing of the managers deep into the child classes. At some point, it would be nice to flatten out the class structure (1 deep) and pass in state from here, similar to Redux.